### ISSUE TITLE
[Research] Using aria-haspopup attribute in the SprkDropdown

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#1329]
**User Story:**
As Spark we want to research whether or not we should be using the aria-haspopup attribute in the SprkDropdown.

**Notes:**
This came out of React Code review 4/22

**AC:**
- whether or not we should be using the aria-haspopup attribute
- No, what are we proposing...
